apps/meteor/ee/server/api/licenses.ts

```diff
@@ -70,6 +78,31 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'licenses.maxGuestUsers',
+	{ authRequired: true },
+	{
+		get() {
+			const maxGuestUsers = getMaxGuestUsers() || null;
+			const activeGuests = Users.getActiveLocalGuestCount();
+
+			return API.v1.success({ maxGuestUsers, activeGuests });
+		},
+	},
+);
+
+API.v1.addRoute(
+	'licenses.maxRoomsPerGuest',
+	{ authRequired: true },
+	{
+		get() {
+			const maxRoomsPerGuest = getMaxRoomsPerGuest() || null;
+
+			return API.v1.success({ maxRoomsPerGuest });
+		},
+	},
+);
+
```

Comment:
any special reason for this information to be publicly available (to any logged in user)? I don't see value making this information public and consider it somewhat confidential.

Reply:
Not exactly :thinking: I decided to create it so that I could use it in the end-to-end tests. Do you think this should be protected by a permission or removed?

Reply:
Fixed! I've removed the endpoints and replaced them in the tests by a constant